custodian/vasp/handlers.py

```diff
@@ -935,62 +934,46 @@ def __init__(self, wall_time=None, buffer_time=300,
                 Should be used with LWAVE = .True. to be useful. If this is
                 True, the STOPCAR is written with LABORT = .TRUE. instead of
                 LSTOP = .TRUE.
-            auto_continue (bool): Use the auto-continue functionality within the
-                VaspJob by ensuring Vasp doesn't delete the STOPCAR
         """
         if wall_time is not None:
             self.wall_time = wall_time
         elif "PBS_WALLTIME" in os.environ:
             self.wall_time = int(os.environ["PBS_WALLTIME"])
         elif "SBATCH_TIMELIMIT" in os.environ:
-            self.walltime = int(os.environ["SBATCH_TIMELIMIT"]) 
+            self.wall_time = int(os.environ["SBATCH_TIMELIMIT"]) 
         else:
             self.wall_time = None
         self.buffer_time = buffer_time
-        self.start_time = datetime.datetime.now()
+        if "JOB_START_TIME" in os.environ:
+            # Enables use of setting JOB_START_TIME as the start
+            # time to use for the walltime handler.  The easiest
+            # way to use this is by executing export JOB_START_TIME=`date`
+            # in the environment (e. g. in a queue submission script)
+            self.start_time = datetime.datetime.strptime(
+                os.environ["JOB_START_TIME"], "%a %b %d %H:%M:%S %Z %Y")
```

Comment:
also, ideally there would be a SLURM/PBS/etc. environment variable that one could use for this, but doesn't seem to be the case (at least for SLURM).